internal/tenancy/tenancytest/namespace_test.go

```diff
@@ -5,11 +5,13 @@ package tenancytest
 
 import (
 	"context"
+	"fmt"
+	"testing"
+
 	"github.com/hashicorp/consul/agent/grpc-external/services/resource"
 	svctest "github.com/hashicorp/consul/agent/grpc-external/services/resource/testing"
 	resource2 "github.com/hashicorp/consul/internal/resource"
```

Comment:
nit: `resource2` seems unusual

```suggestion
	svc "github.com/hashicorp/consul/agent/grpc-external/services/resource"
	svctest "github.com/hashicorp/consul/agent/grpc-external/services/resource/testing"
	"github.com/hashicorp/consul/internal/resource"
```